apps/coscrad-frontend/src/components/tenant/tenant-logo.tsx

```diff
@@ -7,7 +7,7 @@ export const TenantLogo = (): JSX.Element => {
 
     return (
         <StyledImage
-            sx={{ width: { xs: '40px', sm: '55px', md: '61px' } }}
+            sx={{ width: { xs: '30px', sm: '35px', md: '40px' } }}
```

Comment:
My only concern is that we might actually need to inject this in the props if there is a second use case somewhere. Double check with @yaanahuu2 if that's the case.